features/frameworks/utest/utest/utest_case.h

```diff
@@ -111,18 +164,9 @@ namespace v1 {
         bool is_empty() const;
 
     private:
-        const char *description;
-
-        const case_handler_t handler;
-        const case_control_handler_t control_handler;
-        const case_call_count_handler_t repeat_count_handler;
-
-        const case_setup_handler_t setup_handler;
-        const case_teardown_handler_t teardown_handler;
-
-        const case_failure_handler_t failure_handler;
-
+        // see data member in case_t
```

Comment:
I should rephrase the comment and makes it more explicit. 
This class **shall not** declare data members. 

Data members are declared in `case_t` and imported via private inheritance.